src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -920,6 +922,39 @@ void Convolution::addZeroPoints(dnnl::primitive_attr& attr) {
     }
 }
 
+void Convolution::SetPostOpsAndZeroPoints(std::vector<dnnl::primitive_attr> &attrs) {
```

Comment:
To be honest I don't understand why we need so complex solution.
We conditionally initialize attrs[0] with new post ops for brgconv if required.
Here we should initialize legacy post ops and either push_back them to attrs if  shouldTryBrgconv==true or reset attrs[0] otherwise.
Optimizations presented below seems not clear and shouldn't significantly affect FIL.

Reply:
I will try simplify this part.  The goal this part of code is to only create necessary attribute number. So,
  if not having some legacy post ops, only needs 1.
If having dw-conv fused, only needs 1.
Otherwise, if (shouldTryBrgconv || enforceBrgconv), add the second attribute.